Module4.3_Search_and_Sorting_Algorithms/activities/Act1_SearchingAndSorting/11.md

```diff
@@ -0,0 +1,39 @@
+<!--title={Matchsticks to Square: Completing DFS Implementation}-->
+
+After establishing the base cases for our nested `dfs()` algorithm, let's continue onto the rest of the code:
+
+For the matchstick we have selected with the current index, we have four different options, meaning it can be a part of any of the sides of the square, and we will test out these four options by performing recursion.
+
+```python
```

Comment:
code formatting. for code snippets, please don't include indentation if it is not needed for the code to execute. Like I know this line is however many indents within a code block, but if the rest of the block is not in there, do not include the extra indents at the front. lmk if what I wrote was confusing